jekyll/_cci2/deploy-android-applications.adoc

```diff
@@ -0,0 +1,333 @@
+                executor:
+                  name: android/android-docker
+                  tag: "2022.0.7"
+                base64-keystore: BASE64_KEYSTORE
+                release-key-alias: RELEASE_KEY_ALIAS
+                release-key-password: RELEASE_KEY_PASSWORD
+                release-keystore: ./keystore
+                release-store-password: RELEASE_STORE_PASSWORD
+                keystore-properties-working-directory: '.'
+                google-play-key: GOOGLE_PLAY_KEY
+                lane-name: deploy
+                fastlane-working-directory: '.'
+                node-version: "16.13.0"
+                npm-working-directory: '.'
+```
+
+NOTE: The executor has no default value and must be set manually. The executor used must also include Node.**
+
+=== ii. Run each command individually
+
+To run each command individually in your workflow, you will need to add the following commands:
+
+- `decode-keystore`
+- `create-keystore-properties` 
+- `create-google-play-key`
+- `fastlane-deploy` 
+
+Additionally, you need to run either `npm install` or `yarn install` using the Node orb. 
+
+As with the deploy-to-play-store approach, you will not need to set the parameters `base64-keystore`, `release-key-alias`, `release-key-password`, `release-store-password, and `google-play-key`, if you had created environment variables as outlined earlier in this tutorial. 
+
+Below is an example config of this approach for a React Native Project:
+**
+```yaml
+orbs:
+  android: circleci/android@3.0.0
+  node: circleci/node@5.0.2
+jobs:
+  deploy: 
+      docker:
+        - image: cimg/android:2022.07
+      resource_class: large
+      steps: 
+        - checkout
+        - run: 
+            name: clone sample project
+            command: git clone git@github.com:CircleCI-Public/ReactNativeCFD.git
+        - node/install:
+            install-yarn: false
+            node-version: "16.13.0"
+        - run: cd ReactNativeCFD && npm install
+        - android/decode-keystore:
+            keystore-location: android/app/keystore
+        - android/create-keystore-properties:
+            working-directory: android
+        - android/create-google-play-key:
+            working-directory: android
+        - android/fastlane-deploy:
+            working-directory: android
+            lane-name: playstore
+```
+
+== 4.B. Set up config without the Android Orb
+
+. Add the following command to your deployment job in `.circleci/config.yml` to decrypt your keystore from the base64 environment variable set earlier (BASE64_KEYSTORE).
++
+```yaml
+run:
+  name: Decode Android key store
+  command: echo $BASE64_KEYSTORE | base64 -d | tee keystore android/app/keystore > /dev/null
+```
+
+. Next, you need to generate a `keystore.properties` file in order to publish your work to the Google Play Store. 
++
+Before doing that, you need to create a new environment variable named RELEASE_KEYSTORE, that points to the location of the decrypted keystore.
```

Comment:
```suggestion
Before doing that, you need to create a new environment variable named `$RELEASE_KEYSTORE`, that points to the location of the decrypted keystore.
```

Reply:
@devinmarieb ok if I keep the env var name without the backticks and $? Only because that's what I've done for the other env vars that need to be created in the web app, and I don't want to get readers confused and think they need to prefix with the `$` for creating this particular variable.

Reply:
@kbatuigas it's fine for now, but env vars are always typically represented that way, it's not just a CircleCI thing, it's a generic env var syntax thing. It's known the $ is not included, it's what signifies the variable is in fact an environment variable rather than something else. We should update them all at some point. While we are sometimes inconsistent with it, we do represent it that way in most examples we have (an example below)
https://circleci.com/docs/env-vars#setting-an-environment-variable-in-a-project

Reply:
@devinmarieb Yeah, I get the shell syntax... I was thinking specifically with the web app really and how you only need to input the name when creating it. I do agree though that most of our readers probably understand how they're typically represented. I'll go ahead and change the others in the doc.